ethpm/_utils/backend.py

```diff
@@ -30,8 +27,17 @@
     RegistryURIBackend,
 )
 
+try:
```

Comment:
This isn't the most elegant way to handle this but it will only break if an exception is raised in the below methods. I am open to different ideas here.